### ISSUE TITLE
[Major, notification] Notification is not real time update

### ISSUE THREAD

**Issue by oliverchan0222** [skygear-demo/react-chat-demo#28]
1. The notification will be dismissed in few seconds even the users are typing
![screen shot 2017-02-21 at 3 52 36 pm](https://cloud.githubusercontent.com/assets/20634676/23155526/e229877a-f84d-11e6-8e10-8dace9b4519a.png)
2. The notification will not dismiss even the user have deleted all the words
![screen shot 2017-02-21 at 3 54 59 pm](https://cloud.githubusercontent.com/assets/20634676/23155630/717fa7d8-f84e-11e6-8281-5541fc5513ee.png)

**Comment by mayyuen318** [comment]
It takes a little bit time for server to respond?? 
From my testing the delay is 2-3s which is acceptable? 

@rickmak let me know this is normal or is an issue.

**Comment by rickmak** [comment]
No, 2-3 second is not acceptable. But may not related to demo code. Need more inspection.

I suggest to inspect the problem point. To make sure it is back-end performance problem of demo code problem.
